processor/batchprocessor/splitmetrics.go

```diff
@@ -29,12 +29,12 @@ func splitMetrics(size int, src pdata.Metrics) pdata.Metrics {
 
 	src.ResourceMetrics().RemoveIf(func(srcRs pdata.ResourceMetrics) bool {
 		// If we are done skip everything else.
-		if totalCopiedDataPoints == size {
+		if totalCopiedDataPoints >= size {
```

Comment:
This seems like more than a nit change, was this causing an issue before?

Reply:
No issue, it was just me thinking that a correct way is to say ">=".

Reply:
Reverted.